### ISSUE TITLE
[a11y]: ai-chat shoul provide more descriptive labels for main menu and AI menu

### ISSUE THREAD

**Issue by Daniel-Schulz-Private** [carbon-design-system/carbon-labs#912]
### Package version

0.18.0

### React version

18.3.1

### Automated testing tool and ruleset

n/a

### Assistive technology

JAWS

### Description

The existing Label "Open" (of the main menu of ai-chat) and the non-existing label of the AI menu are too vague or insufficiently descriptive, to explain the purpose of the controls.

Recommended fix: Ensure that all interactive controls have specific and descriptive labels that accurately communicate their function or the information required.
Recommend to provide a Descriptive Visual Label which explain the purpose of the control , and associate the same with Element by using HTML Label for attribute or ARIA Labeled by.

User impact: When controls have unclear or non-descriptive labels, users—particularly those using assistive technologies—may misunderstand the purpose of the control,
leading to errors, hesitation, or inability to complete tasks.

<img width="707" height="221" alt="Image" src="https://github.com/user-attachments/assets/1fa27ab2-5b15-4095-b8b7-9d8b80474803" />

### WCAG 2.1 Violation

WCAG Mapping: 2.4.6 Headings and Labels

### Reproduction/example

n/a

### Steps to reproduce

1. open ai-chat
2. observer vague labels of both main menu (top-left hand side) and AI menu (top right hand side)

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-labs/blob/main/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/carbon-labs/issues) for duplicate problems
